src/lib/core/testing/dispatch-events.ts

```diff
@@ -0,0 +1,26 @@
+import {
+  createFakeEvent,
+  createKeyboardEvent,
+  createMouseEvent,
+  createTransitionEndEvent
+} from './event-objects';
+
+/** Shorthand to dispatch a fake event on a specified node. */
+export function dispatchFakeEvent(node: Node, eventName: string) {
+  node.dispatchEvent(createFakeEvent(eventName));
+}
+
+/** Shorthand to dispatch a keyboard event with a specified key code. */
+export function dispatchKeyboardEvent(node: Node, keyCode: number, type = 'keydown') {
+  node.dispatchEvent(createKeyboardEvent(type, keyCode));
+}
+
+/** Shorthand to dispatch a mouse event on the specified coordinates. */
+export function dispatchMouseEvent(node: Node, type: string, x = 0, y = 0) {
+  node.dispatchEvent(createMouseEvent(type, x, y));
+}
+
+/** Shorthand to dispatch a transition event with a specified property. */
+export function dispatchTransitionEndEvent(node: Node, propertyName: string, elapsedTime = 0) {
```

Comment:
Ah weird, I thought `transitionstart` would also be valid, but the docs do say just `transitionend`